src/app.controller.ts

```diff
@@ -9,4 +9,12 @@ export class AppController {
   getHello(): string {
     return this.appService.getHello();
   }
+
+  @Post('bad-format')
+  async bad_format(@Body( ) data: any)
+  {
+    console . log ( 'this should not be legal');
```

Comment:
Codacy has a fix for the issue: [Replace `·.·log·(·` with `.log(`](https://app.codacy.com/gh/darklam/code-review-testing/pullRequest?prid=8068379)
```suggestion
    console.log('this should not be legal');
```